voctopublish/model/ticket_module.py

```diff
@@ -158,6 +158,16 @@ def _validate_(self, key, optional=False):
                 raise TicketException(key + ' is missing in ticket')
         return value
 
+    def get_raw_property(self, key, optional=True):
+        value = None
+        if key in self.__tracker_ticket:
+            value = self.__tracker_ticket[key]
+        else:
+            if not optional:
+                logging.debug(key + ' is missing in ticket')
```

Comment:
This condition causes an exception and should therefore not be logged at debug level but error or something like that.